Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView.swift

```diff
@@ -161,12 +152,9 @@ open class ChatChannelAvatarView: _View, ThemeProvider, SwiftUIRepresentable {
         }
 
         var combinedImage: UIImage?
-
+        let images = avatars
         let imageProcessor = components.imageProcessor
-
-        let images = avatars.map {
-            imageProcessor.scale(image: $0, to: components.avatarThumbnailSize)
-        }
```

Comment:
`avatars` contains downloaded and resized images + placeholders. Therefore, no point of resizing downloaded image again. Placeholder resizing happens above.